NOTES.UNIX

```diff
@@ -8,35 +8,53 @@
  Shared libraries and installation in non-standard locations
  -----------------------------------------------------------
```

Comment:
I'd like to see explicit note that OpenSSL doesn't set shared library search path anymore. Somewhere early, maybe even as opening sentence, so that user understands why this note has to be read.

Reply:
I'll skip "any more" and simply state that it doesn't.  I don't think this is the place for a historical record.